From d61dad05a868109b5307d2184818dcbde4220d26 Mon Sep 17 00:00:00 2001
From: neovali <valigarmanda55@gmail.com>
Date: Sun, 12 Nov 2023 23:37:15 +0800
Subject: [PATCH] stop spamming logs

---
 UI/framelessdialog.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/UI/framelessdialog.cpp b/UI/framelessdialog.cpp
index def9268..dba088b 100644
--- a/UI/framelessdialog.cpp
+++ b/UI/framelessdialog.cpp
@@ -593,7 +593,7 @@ void CFramelessDialog::mouseMoveEvent(QMouseEvent *e)
         top = qAbs(y - r.top()) <= 5;
         bottom = qAbs(y - r.bottom()) <= 5;
         bool hor = left | right, ver = top | bottom;
-        qDebug() << "l,r,t,b: " << left << right << top << bottom;
+        // qDebug() << "l,r,t,b: " << left << right << top << bottom;
 
         if(hor && bottom)
         {
-- 
2.41.0

